packages/node/src/utils/project.ts

```diff
@@ -299,3 +310,57 @@ export function loadChainTypesFromJs(
   }
   return rawContent;
 }
+
+export async function getExistingProjectSchema(
```

Comment:
Could this and the below functions live in node-core? It would be handy for other chains to not duplicate this